alephclient/tasks/crawldir.py

```diff
@@ -2,24 +2,30 @@
 import threading
 from queue import Queue
 from pathlib import Path
+from typing import Optional, Dict
 
 from alephclient import settings
+from alephclient.api import AlephAPI
 from alephclient.errors import AlephException
 from alephclient.util import backoff
 
 log = logging.getLogger(__name__)
 
 
-def _get_foreign_id(root_path, path):
+def _get_foreign_id(root_path: Path, path: Path) -> Optional[str]:
     if path == root_path:
         if path.is_dir():
-            return
+            # NOTE: Raise exception
```

Comment:
I think we should raise an exception there.

Reply:
Nope, this is not an error state.

Reply:
:+1: